src/System.CodeDom/tests/Compiler/CodeDomProviderTests.cs

```diff
@@ -0,0 +1,539 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.CSharp;
+using Microsoft.VisualBasic;
+using Xunit;
+
+namespace System.CodeDom.Compiler.Tests
+{
+    public class CodeDomProviderTests
+    {
+        [Fact]
+        public void GetAllCompilerInfo_ReturnsMinimumOfCSharpAndVB()
+        {
+            CompilerInfo[] compilerInfos = CodeDomProvider.GetAllCompilerInfo();
+            Assert.True(compilerInfos.Length >= 2);
+            Assert.Equal(typeof(CSharpCodeProvider), compilerInfos[0].CodeDomProviderType);
+            Assert.Equal(typeof(VBCodeProvider), compilerInfos[1].CodeDomProviderType);
```

Comment:
Nit: I don't think we guarantee an order... probably best to use Contains or something like that.